<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>58</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>606</width>
    <height>58</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>606</width>
    <height>58</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ADPAFC</string>
  </property>
  <widget class="QPushButton" name="butVar1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вариант 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butVar2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вариант 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butVar3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вариант 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butVar4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вариант 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butVar5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вариант 5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
